### PR TITLE
MDS-4357: Create an overview page in Minespace for Projects

### PR DESCRIPTION
**Pull request by BenHarrington**
## Objective 

[MDS-4357](https://bcmines.atlassian.net/browse/MDS-4357)

- Repurposing project summaries landing to projects landing
- Create new route component Project(Project Overview)

### CODE REVIEW

File: services/minespace-web/common/actionCreators/projectActionCreator.js [hunk 1]
```diff
@@ -132,12 +132,43 @@ export const fetchProjectSummaries = (payload) => (dispatch) => {
...
+
+export const fetchProjectById = (projectGuid) => (dispatch) => {
+  dispatch(request(reducerTypes.GET_PROJECT));
+  dispatch(showLoading());
+  return CustomAxios({ errorToastMessage: Strings.ERROR })
+    .get(ENVIRONMENT.apiUrl + API.PROJECT(projectGuid), createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_PROJECT));
+      dispatch(projectActions.storeProject(response.data));
+      dispatch(success(reducerTypes.GET_PROJECT_SUMMARY));
```
Comment (areyeslo) [inline services/minespace-web/common/actionCreators/projectActionCreator.js:162]: So far, I have not found any comments. Just curious on this one:
Why this is not added Project summary fetchProjectsByMine as in here?

File: services/minespace-web/src/components/dashboard/mine/projects/Projects.js [hunk 2]
```diff
@@ -147,31 +126,17 @@ export class ProjectSummaries extends Component {
...
-                    <Button type="primary" disabled>
-                      <PlusCircleFilled />
-                      Start a new application
-                    </Button>
-                  </Tooltip>
-                )}
+                <Button type="primary">
+                  <PlusCircleFilled />
+                  Create New Project
+                </Button>
```
Comment (areyeslo) [inline services/minespace-web/src/components/dashboard/mine/projects/Projects.js:132]: Are we missing the validation to have just one project description?

Comment (BenHarrington) [reply services/minespace-web/src/components/dashboard/mine/projects/Projects.js:132]: This is now the Project space and a mine can have as many projects as it wants. Additionally on the ProjectOverview page there is no way to manually create a new project summary so we should be secure on the frontend.

File: services/minespace-web/common/actionCreators/projectActionCreator.js (hunk 1 above)
Comment (BenHarrington) [reply services/minespace-web/common/actionCreators/projectActionCreator.js:162]: FetchProjectsByMine returns a list of projects and the summaries are contained within them, however when using fetchProjectById it returns a single entity and it is easier to map both the project and project_summary in our Redux store.

The component state needed by the Project list component shouldn't require us to populate the PROJECT_SUMMARIES store.

Comment (areyeslo) [reply services/minespace-web/common/actionCreators/projectActionCreator.js:162]: Thanks for clarifying :)

**Review by areyeslo — DISMISSED** [review]
Excellent job on this Ben!!. Ship it to Prod!! :rocket:

File: services/minespace-web/src/styles/overrides/antd-overrides.scss [hunk 3]
```diff
@@ -249,17 +249,17 @@ h5.ant-typography,
   border-bottom: 2px solid $primary-color;
   border-radius: 0;
   color: $primary-color;
-  text-transform: uppercase;
+  // text-transform: uppercase;
```
Comment (matbusby-fw) [inline services/minespace-web/src/styles/overrides/antd-overrides.scss]: Are these being re-added at some point, or can they be deleted?

**Review by matbusby-fw — DISMISSED** [review]
Looks good to me.  I don't have all the context regarding the projects, but from what I can see everything looks good.

Left one comment regarding commented out styles.

**Review by areyeslo — DISMISSED** [review]

**Review by areyeslo — APPROVED** [review]

**Pull request merged**